app/config/locale/terms.json

```diff
@@ -406,7 +406,7 @@
     },
     {
         "term": "countries.fr",
-        "comment": "These country names are used in Appwrite's Locale API. Learn more at https://appwrite.io/docs/client/locale#localeGetCountries",
+        "comment": "Ces noms de pays sont utilisés dans l'API Locale d'Appwrite. Pour en savoir plus, consultez le site https://appwrite.io/docs/client/locale#localeGetCountries.",
```

Comment:
You don't need to modify this file as part of the PR. :slightly_smiling_face:  You can revert this change